### DISCUSSION TITLE
Is there a text area element?

### DISCUSSION THREAD

**Discussion by 200729** [zauberzeug/nicegui#27] [Q&A] [answered]
Is there text area element? (multiline)

**Comment by falkoschindler** [comment]
Yes, there is.

Since NiceGUI uses the Quasar framework for most input elements, you can create a [text area](https://quasar.dev/vue-components/input#textarea) setting the "type" as follows:

```python
ui.input('Hello world!').props('type=textarea')
```

**Reply by 200729** [reply]
It is working, thank you!

**Comment by falkoschindler** [comment, marked as answer]
Since NiceGUI 1.1.9 there is `ui.textarea`.
